kyo-prelude/shared/src/main/scala/kyo/Local.scala

```diff
@@ -23,8 +23,8 @@ abstract class Local[A]:
       * @return
       *   An effect that produces the current value
       */
-    final def get(using Frame): A < Any =
```

Comment:
These weren't inlined I believe because of compilation issues since we typically inline suspension methods. I've made `State` public and was able to inline the methods.